python2/httplib2/__init__.py

```diff
@@ -1615,6 +1624,7 @@ def __init__(
         ca_certs=None,
         disable_ssl_certificate_validation=False,
         ssl_version=None,
+        enable_cookie=True
```

Comment:
Please name it `cookies` or `cookie_jar` and accept an object with `get/set/pop/clear` methods, to allow external or persistent storage. With that `Http(cookies={})` would implement the same functionality as current code, extensible and all for minimal changes.

Reply:
Now I name it as cookie_jar.  It accepts a httplib2.CookieJar object.